crates/db_schema/src/impls/comment.rs

```diff
@@ -17,20 +17,6 @@ use diesel_ltree::Ltree;
 use url::Url;
 
 impl Comment {
-  pub fn permadelete_for_creator(
-    conn: &mut PgConnection,
-    for_creator_id: PersonId,
-  ) -> Result<Vec<Self>, Error> {
-    use crate::schema::comment::dsl::*;
-    diesel::update(comment.filter(creator_id.eq(for_creator_id)))
-      .set((
-        content.eq("*Permananently Deleted*"),
```

Comment:
This section still needs to be here, especially the `deleted.eq(true)`. You just don't need to overwrite the content.

Otherwise all their posts and comments would still be publicly visible.